test/parallel/test-whatwg-encoding-custom-textdecoder.js

```diff
@@ -51,6 +51,20 @@ assert(TextDecoder);
   });
 }
 
+// Invalid encoders
+{
+  ['meow', 'nonunicode', 'foo', 'bar'].forEach((fakeEncoding) => {
+    assert.throws(
+      () => { new TextDecoder(fakeEncoding); },
+      {
+        code: 'ERR_ENCODING_NOT_SUPPORTED',
+        name: 'RangeError',
+        message: `The "${fakeEncoding}" encoding is not supported`
```

Comment:
nit: We try to avoid testing the exact message when there is a code. If you want to validate that the passed encoding is part of the message, you can use a RegExp

Reply:
Didn't know that! Thanks for the heads-up!